content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -252,7 +258,7 @@ configuration parameters from your AppID:
     * `oAuthServerUrl`
     
 7. Register the Kubeflow OIDC redirect page. The Kubeflow OIDC redirect URL is `http://<kubeflow-FQDN>/login/oidc`. 
-`<kubeflow-FQDN>` is the endpoint for accessing Kubeflow. By default, the `<kubeflow-FQDN>` on IBM Cloud is `<worker_node_external_ip>:31380`.
+`<kubeflow-FQDN>` is the endpoint for accessing Kubeflow. By default, the `<kubeflow-FQDN>` on IBM Cloud is `<worker_node_external_ip>:31380`. If you don't have any experience on Kubernetes, you can [Expose the Kubeflow endpoint as LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer) and use the **EXTERNAL_IP** for your `<kubeflow-FQDN>`.
```

Comment:
I'm not well-versed in KF and K8s, so I apologize for asking this - can you help explain what you mean by "if you don't have any experience on Kubernetes"? Does it mean - the user may not know how to work with K8s or is it something else?

Reply:
We are seeing some users with zero Kubernetes background don't understand what is a worker node and why it has to use a port number. So we are recommending those users to enable load balancer for Kubeflow.

There's limited number of free load balancers on the IBM Cloud cluster, so we are not enforcing load balancer on the default Kubeflow deployment.